cni/pkg/repair/repair.go

```diff
@@ -125,8 +127,8 @@ func (bpr BrokenPodReconciler) labelBrokenPod(pod v1.Pod) (err error) {
 		log.Errorf("Failed to update pod: %s", err)
 		return err
 	}
-	bpr.Metrics.PodsRepaired.Increment()
-	return err
+	podsRepaired.With(typeLabel.Value(labelType)).Increment()
+	return nil
```

Comment:
nit: should you change line 126 to be `:=` instead of `=` as well and remove the `(err error)` return?

Reply:
Not sure I follow. I removed `err` or `nil` from `return` statement, and I think it is cleaner now.

Please take another look. I did a bit of more change to include a new result tag. thanks!